tests/test_codecs/test_blosc.py

```diff
@@ -54,3 +57,29 @@ async def test_blosc_evolve(store: Store, dtype: str) -> None:
         assert blosc_configuration_json["shuffle"] == "bitshuffle"
     else:
         assert blosc_configuration_json["shuffle"] == "shuffle"
+
+
+async def test_typesize() -> None:
+    a = np.arange(1000000)
```

Comment:
```suggestion
    a = np.arange(2**16, dtype=np.uint16)
```
As a thought, worth explicitly specifying the data type (and making the data smaller)? Don't know if it will fix the windows issue, but I think worth doing anyway os there's a concrete bytesize, and perhaps using integer data type will help with linux/windows because perhaps they have different floating point implementations (although that's wild speculation on my part...)

Reply:
`arange` is default `uint64` so I'll push something with that added.